src/wasm/wasm-s-parser.cpp

```diff
@@ -1893,18 +1879,7 @@ void SExpressionWasmBuilder::parseImport(Element& s) {
       throw ParseException("invalid import");
     }
   }
-  if (!s[i]->quoted()) {
-    if (s[i]->str() == MEMORY) {
-      kind = ExternalKind::Memory;
-    } else if (s[i]->str() == TABLE) {
-      kind = ExternalKind::Table;
-    } else if (s[i]->str() == GLOBAL) {
-      kind = ExternalKind::Global;
-    } else {
-      throw ParseException("invalid ext import");
-    }
-    i++;
```

Comment:
This belongs to `parseImport`. This part was first added in https://github.com/WebAssembly/binaryen/commit/793863a5d0e2f864f46dac86baa8e12c63b5b004, and no test case was provided with the commit, so I can't guess what a test case that triggers this part of code would look like. Could you provide an example?